src/conditionals/simple_programs_with_decision_points/hard/q8/MainTest.java

```diff
@@ -0,0 +1,63 @@
+package conditionals.simple_programs_with_decision_points.hard.q8;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+
+    static final int THRESHOLD = 50;
+
+    @Override
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "1\n2";
+
+        return new Clause[]{
+            new StringLiteral("Enter x: "),
+            new Optional(new StringLiteral(" ")),
+            new NewLine(),
+            new StringLiteral("Enter y: "),
+            new Optional(new StringLiteral(" ")),
+            new NewLine(),
+            new PlaceHolder()
+        };
+    }
+
+    static Stream<Arguments> inputProvider() {
+        return Stream.of(
+            // Case 1: Sum is less than the threshold
+            Arguments.of(1, 10, "Sum is less than " + THRESHOLD),
+            // Case 2: Sum is less than the threshold
+            Arguments.of(25, 24, "Sum is less than " + THRESHOLD),
```

Comment:
you don't have to change this if you don't want to but an example for how a threshold variable is typically used would be instead of having 25 and 24, have THRESHOLD/2 and THRESHOLD/2 - 1 so when you change up the number of the threshold on the website you don't have to add additional variables for 25, 24, and 26